staff/ana-cano/precourse/airlines/index.js

```diff
@@ -0,0 +1,71 @@
+//airlines 
+var flight = [
+    { id: 00, to: 'Bilbao', from: 'Barcelona', cost: 1600, scale: false },
+    { id: 01, to: 'New York', from: 'Barcelona', cost: 700, scale: false },
+    { id: 02, to: 'Los Angeles', from: 'Madrid', cost: 1100, scale: true },
+    { id: 03, to: 'Paris', from: 'Barcelona', cost: 210, scale: false },
+    { id: 04, to: 'Roma', from: 'Barcelona', cost: 150, scale: false },
+    { id: 05, to: 'London', from: 'Madrid', cost: 200, scale: false },
+    { id: 06, to: 'Madrid', from: 'Barcelona', cost: 90, scale: false },
+    { id: 07, to: 'Tokyo', from: 'Madrid', cost: 1500, scale: true },
+    { id: 08, to: 'Shangai', from: 'Barcelona', cost: 800, scale: true },
+    { id: 09, to: 'Sydney', from: 'Barcelona', cost: 150, scale: true },
+    { id: 10, to: 'Tel-Aviv', from: 'Madrid', cost: 150, scale: false }];
+
+    // 1 Preguntar nombre usuario y dar Bienvenida
+function airlinesSkylab() {

```

Comment:
Para que el código sea más legible, intenta dar a esta función un nombre que describa mejor lo que hace :)

Reply:
I realized the change, thank you